### PR TITLE
add: service that gives you the country code for a given timezone

### PR DESCRIPTION
**Pull request by micou**
New service created called CountryCodeLookup.
Ppl might want to convert their timezone into a country ISO "ALPHA-2 Code.
A function has been created to know whether or not your timezone is in europe. (based on https://gist.github.com/henrik/1688572)

### CODE REVIEW

File: src/test/java/com/databerries/timezone/CountryCodeLookupTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    public void testShouldBeInEurope() {
+        boolean isInEurope = countryCodeLookup.isTimezoneInEU("Europe/Paris");
+        Assert.assertThat(isInEurope, is(true));
+    }
+
+    @Test
+    public void testShouldNotBeInEurope() {
+        boolean isInEurope = countryCodeLookup.isTimezoneInEU("America/New_York");
+        Assert.assertThat(isInEurope, is(false));
+    }
```
Comment (alexiswoo) [inline src/test/java/com/databerries/timezone/CountryCodeLookupTest.java]: should we add a test for the `getCountryCode` method ?

File: src/main/resources/eu_country_code.csv [hunk 2]
```diff
@@ -0,0 +1,65 @@
+country code,description
```
Comment (alexiswoo) [inline src/main/resources/eu_country_code.csv]: could be easier to read if we sort it either by country code or by formal name ?

**Review by alexiswoo — APPROVED** [review]

File: src/main/resources/eu_country_code.csv (hunk 2 above)
Comment (micou) [reply src/main/resources/eu_country_code.csv]: you right !

File: src/test/java/com/databerries/timezone/CountryCodeLookupTest.java (hunk 1 above)
Comment (micou) [reply src/test/java/com/databerries/timezone/CountryCodeLookupTest.java]: you right :+1:

**Comment by alexiswoo** [comment]
:+1:

**Pull request merged**